client/src/pages/SignUp.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Container from "../components/Container";
+import Item from "../components/Item";
+
+/* ---------- MATERIAL UI ---------- */
+// import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import { Typography } from "@material-ui/core";
```

Comment:
These can be combined as `import { Button, Typography } from "@material-ui/core"`. When we have many more imports this will help keep it clean